core/store/src/db.rs

```diff
@@ -462,6 +462,7 @@ fn rocksdb_options() -> Options {
     opts.set_keep_log_file_num(1);
     opts.set_bytes_per_sync(1048576);
     opts.set_write_buffer_size(1024 * 1024 * 512 / 2);
+    opts.set_db_write_buffer_size(1024 * 1024 * 64);
```

Comment:
is this the limit on the total amount of column buffers or is this a separate buffer?